src/libos/crates/sworn-disk/README.md

```diff
@@ -0,0 +1,185 @@
+## JinDisk (SwornDisk)
```

Comment:
I appreciate the effort that you have been put into writing this README full of technical details. But I would highly suggest removing most of them. This is because this README reads like a short paper (or an extended abstract), but it should have been read like a project README. The two types of documents serve different audience, thus having different structures and focuses. A paper emphasizes technical novelties and intern workings (this is what the current README does). A project README focuses on selling the project to its (potential) users, explaining why it is useful and how to use. See the [README of Linux's version of JinDisk](https://github.com/jinzhao-dev/jindisk). And since this main user is Occlum, we don't even need to sell. In addition, we will soon make the research paper available on Arxiv.org, before its official publication. The paper should be the primary source for technical details. After the Arxiv version is available, we will add a link to the paper in this project README.

In short, keep this README short and concise and focus on providing information that is absent in the research paper.